storage/google/cloud/storage/_signing.py

```diff
@@ -80,17 +86,34 @@ def get_signed_query_params(credentials, expiration, string_to_sign):
     }
 
 
-def get_expiration_seconds(expiration):
+def get_expiration_seconds_v2(expiration, max_age):
```

Comment:
Remove `max_age` until we get feedback that it's necessary.

Reply:
OK. c37e77a